### PR TITLE
OWASP SAMM collaboration meeting and Crowdin introduction

### PR DESCRIPTION
**Pull request by martinmarsicano**

### CODE REVIEW

File: tab_eventos.md [hunk 1]
```diff
@@ -7,6 +7,31 @@ tags: Uruguay
 ---
 # Eventos
 
+## 2021
+
+### Jornada de Colaboración - AntiSamy
```
Comment (fzipi) [inline tab_eventos.md]: I believe here we should say `SAMM`, right?

**Review by fzipi — CHANGES_REQUESTED** [review]
We just need to change the Antisamy for SAMM, and will be fine.

File: tab_eventos.md (hunk 1 above)
Comment (martinmarsicano) [reply tab_eventos.md:12]: Yes,you're right.

**Review by martinmarsicano — COMMENTED** [review]
Yes, that's correct

**Review by fzipi — APPROVED** [review]
LGTM

**Pull request merged**